_posts/2023-06-04-not-generating-constructors.md

```diff
@@ -0,0 +1,432 @@
+---
+tags:
+- Subspace
+- C++
+- Performance
+- Memory Safety
+---
+
+# Avoiding Death by a Thousand Constructors

```

Comment:
I think there might be a more direct way of stating the headline and summary. Something along the lines of "efficient and safe" or "concise codegen with safety" or "safety without performance compromise" or such.

Reply:
I will sleep on this title and see what I dream up.

Reply:
I came up with "Bounds Safety: Avoiding Death by a Thousand Constructors"